lib/sms/sms.c

```diff
@@ -65,11 +71,12 @@ static struct sms_subscriber subscribers[CONFIG_SMS_SUBSCRIBERS_MAX_CNT];
 /* AT monitors for received SMS messages (CMT) and status reports (CDS). */
```

Comment:
Looks good. Just for the sake of completeness, you could also mention CMS in the comment and open the abbreviation.

Reply:
Good that you noticed, I intended to do that by forgot about it.